chatlearn/schedule/model_manager.py

```diff
@@ -190,8 +190,7 @@ def sync_parameters(self, episode_offset=0, requires_grad=None, validate=False):
     def set_func_decorator(self, model):
         if is_decorated(model.name):
             return
-        # decorate async method here will raise cannot serialize coroutine object error
-        call_funcs = model.call_funcs if not isinstance(model, VLLMModuleV2) else []
+        call_funcs = model.call_funcs
```

Comment:
Is the `time_summary` correct after changing this line of code? The comment here indicates the previous situation.

Reply:
time_summary is ok when we using sync_engine, unrelated to this pr.